src/en/proposals/fairlysadpanda-revolutionaries.md

```diff
@@ -0,0 +1,333 @@
+
+Clowns get the ability to put up annoying "Visit Clown Planet" tourism posters.
+
+Posters can be be destroyed by spraying them with paint or tearing them down.
+
+### Addition of Loyalty Loudspeakers in Sec
+
+These Loyalty Loudspeakers are audio emitters that reduce a player’s Influence level, even if they’re converted. If a player who is converted de-converts, their Influence immediately drops to zero. This is a new machine that requires quite a bit of power to run. A button is added to the Warden’s office to disable and enable all loudspeakers. The loudspeakers play calming, smooth jazz.
+
+These Loudspeakers cannot be made on the station. One of the key jobs of Sec is to ensure Sec remains safe; if it's been bombed and the Loudspeakers are inoperable, the Revolution is succeeding.
+
+### Head Revs can use an implant ability to silently convert characters with high enough Influence to being a Rev.
+
+This replaces flashing outright, and makes conversion a stealthy game. Revs do not know who the Head Revs are unless the Head Rev outs themselves.
+
+### Add the Interrogator, an electric chair that tests Influence levels
+
+The Interrogator is a high-power-draw device that lives in Sec. This machine does three things:
+
+1. Minorly fries the person placed inside it. `Your hair stands on end!` If you're a Diona, you are set on fire. `The lightning ignites your limbs!` It does no damage to Slimes. `You feel tingly`.
+2. Reads their Influence level.
+3. Causes the lights to flicker station-wide.
+
+This allows Sec to ensure anyone they're releasing from prison has a low influence level.
+
+It can be emagged to dramatically increase the power draw, EMPing the station for a period of time, and instantly frying any poor sod put inside it.
+
+### Add the Centcomm Uplink Terminal as an indestructable core console on the Bridge.
+
+This terminal is the core security uplink between the space station and the rest of the universe (in fiction). It can send announcements and notifies who is the current chain-of-command.
+
+This console is indestructable, protected by military-grade force fields, immovable and powered by a nuclear engine that will outlast the star it's anchored to.
+
+This is the end-game objective that Revs want to compromise.
+
+### Denote the IDs that Heads Of Staff spawn with as Head Of Staff IDs
+
+These are the mid-game objectives that Revs want to obtain. Because these are stealable objects, this gives Rev Thiefs a unique role.
+
+### Killing the Heads is now not required for the Revs to win, instead requiring taking the Bridge and compromising the Centcomm Uplink Terminal and defending the Terminal for a period of time.
+
+This removes one half of the round-removal problem and gives the Revs a clear final target that works a bit like the Nukie bomb.
+
+To compromise the terminal, the Revs need access to a Head Of Staff ID. Any will do: using them on the terminal allows for the Uplink to be disabled after a timer.
```

Comment:
How does one disarm such event?

Reply:
Also might suggest needing something more than one head-level ID and the bridge because that'd still be a quick way for any traitor to fuck with the WHOLE station or nukies to deviate from their actual objective? Perhaps some flavourtext about the HR implant being what enables the process and the ID only being needed to authorise it.

Reply:
Maybe. I do want the Revs to be able to end the round if a HR is alive somewhere (but captured, maybe) so it'd be any Rev able to do it, but that might be metagamey.

I'll break the round ending mechanic into its own PR so it can be discussed, as it's the first major step and is already a rework to the mode ("Revs 1.5" if you will)

Reply:
If the head rev is captured that should naturally cause it to be a revolutionry failure. They should be working their hardest to get their leader back before they're executed for their crimes. Having non HRs get game ending power would make it all to easy to be abused.